staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/object_count_tracker.go

```diff
@@ -148,6 +140,19 @@ func (t *objectCountTracker) Get(groupResource string) (int64, error) {
 	return 0, ObjectCountNotFoundErr
 }
 
+// RunUntil start all the necessary maintenance.
+func (t *objectCountTracker) RunUntil(stopCh <-chan struct{}) {
+	go func() {
```

Comment:
maybe the `RunUntil` should block and be the caller the one that spawns it in a goroutine?


```
go t.RunUntil(stopCh)
```

I was trying to find if there is a pattern already but didn't find a clear indicator